internal/deviations/README.md

```diff
@@ -1,6 +1,22 @@
-## Guidelines to add deviations to FNT tests
+# Guidelines to add deviations to FNT tests
 
-### Adding Deviations
+## When to use deviations
+
+1. Deviations may be created to use alternate OC or use CLI instead of OC to achieve the operational intent described in the README.
```

Comment:
Can you a few examples on how to use deviations in code:

Using alternate OC
```go
...
conf := configureDUT(dut) // returns *oc.Root

if deviations.AlternateOCEnabled(t, dut) {
    conf.SetAlternateOC(val)
} else {
    conf.SetRequiredOC(val)
}
...
```

Using vendor CLI instead of OC
```go
...
conf := configureDUT(dut) // returns *oc.Root

if deviations.RequiredOCNotSupported(t, dut) {
    switch dut.Vendor() {
    case ondatra.JUNIPER:
        configureDeviceUsingCli(t, dut, vendorConfig)
    }
}
...
```

Reply:
Done